packages/e2e-tests/tests/logpoints-01.test.ts

```diff
@@ -47,6 +50,14 @@ test(`logpoints-01: log-points appear in the correct order and allow time warpin
   await expect(logPointMessages.first()).toHaveText(/Beginning/);
   await expect(logPointMessages.last()).toHaveText(/Ending/);
 
+  await setLogpointBadge(page, logPointMessages.nth(1), "unicorn");
+  await waitFor(async () => {
+    await verifyLogpointBadge(logPointMessages.nth(1), "unicorn");
+    await verifyLogpointBadge(logPointMessages.nth(2), "unicorn");
+  });
+  await verifyLogpointBadge(logPointMessages.first(), null);
+  await verifyLogpointBadge(logPointMessages.last(), null);
```

Comment:
I like the idea of adding test coverage for this! I'm not sure this test is actually verifying that the setter works though, since it doesn't check the before state. Can we add an explicit check for that too?